pkgs/by-name/sy/syncrclone/package.nix

```
@@ -0,0 +1,31 @@
+{ lib
+, python3
+, fetchFromGitHub
+}:
+
+python3.pkgs.buildPythonApplication {
+  pname = "syncrclone";
+  version = "20230310.0.BETA";
```

```suggestion
  version = "unstable-2023-03-23";
```

This is the exact date the rev 137c9c4cc737a383b23cd9a5a21bb079e6a8fc59 was released.